desktop/test/e2e/specs/login.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-console, import/no-nodejs-modules */
+const { createWriteStream } = require( 'fs' );
+const { mkdir } = require( 'fs/promises' );
+const path = require( 'path' );
+const { _electron: electron } = require( 'playwright' );
+
+// Linux:
+const APP_PATH = path.join( __dirname, '../../../release/linux-unpacked/wpcom' );
+// Mac:
+// const APP_PATH = path.join(
```

Comment:
If testing locally on Mac, you can replace the `APP_PATH` argument with this one.